### ISSUE TITLE
Separator is not working correctly

### ISSUE THREAD

**Issue by valerybugakov** [scmbreeze/scm_breeze#298]
Hello, first of all, thank you for the amazing scripts! Love them.

It looks like the output from the scripts is not separated via the separator symbol. 

<img width="881" alt="Screenshot 2020-02-03 at 16 02 28" src="https://user-images.githubusercontent.com/3846380/73639262-97931f80-469e-11ea-8d07-6b398ea762f0.png">

Instead of printing just `/Users/val/.dotfiles/oldschool/dots/.eslintrc` it prints all the changed files concatenated via `|`. I've tried reinstalling scripts but still get the same issue. What may be the cause of such behavior?

**Comment by eballerini** [comment]
I am running into the same problem. I am not sure what's changed but I have 2 MacBookPros.
It's working on this MBP:
 - Mojave 10.14.1
 - iterm2 3.3.6
 - zsh 5.8 (x86_64-apple-darwin18.7.0) 
It's not working on that MBP:
 - Catalina 10.15.7
 - iterm2 3.4.3
 - zsh 5.8 (x86_64-apple-darwin19.3.0)
 - ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]

**Comment by valerybugakov** [comment]
Yep, still have this issue. Would appreciate any insight on how to fix it.
